### PR TITLE
Update docker-compose.yaml README.md and backend

### PR DESCRIPTION
**Pull request by Raynur2000**

### CODE REVIEW

**Comment by Raynur2000** [comment]
@aabounegm I changes Mongo DB into Postgres DB in docker-compose.yaml

File: docker-compose.yaml [hunk 1]
```diff
@@ -34,20 +34,18 @@ services:
       - ./frontend:/app/    # To detect changes. Dev only
       - /app/node_modules   # Except for node_modules
   db:
-    image: mongo:4.2
+    image: postgres:12.0-alpine
+    volumes:
+      - database:/var/lib/postgresql/data/
     env_file:
-      - ./mongo.env
+      - ./backend/.env
```
Comment (aabounegm) [inline docker-compose.yaml:41]: This should actually be a separate `.env`, as in the example you modified in the README. It is not the same as the backend's `.env` file.

Also, if you want to expose its port for dev reasons, it should be `5432` by default

Comment (Raynur2000) [reply docker-compose.yaml:41]: Do I have to write expose: -5432?

Comment (aabounegm) [reply docker-compose.yaml]: It's
```yaml
ports:
- 5432:5432
```
and you don't _have_ to write it, I am just recommending it during development for easier debugging with tools like pgAdmin

Comment (Raynur2000) [reply docker-compose.yaml]: Got it! Thanks

**Pull request merged**